dvc/fs/s3.py

```diff
@@ -20,6 +21,19 @@
 _AWS_CONFIG_PATH = os.path.join(os.path.expanduser("~"), ".aws", "config")
 
 
+def process_ssl_verify_param(ssl_verify_config_value):
+    """
+    Checks the type of the input parameter and returns the
+    boolean or certPath based on the input
+    """
+    if isinstance(ssl_verify_config_value, bool):
+        return ssl_verify_config_value
+    try:
+        return strtobool(ssl_verify_config_value)
+    except ValueError:
+        return ssl_verify_config_value
```

Comment:
This should be on the config level, like `supported_cache_types` we've mentioned before. str to bool conversions don't belong on the fs level. If you don't want to mess around with it more - reverting to supporting only bool would be totally acceptable.

Reply:
Just going to try to implement it, otherwise I'll revert

Reply:
@efiop Could you show me where the value of supported_cache_type is used? I can't seem to find it anywhere

Reply:
Done